.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
Instead of picking the latest stable version within the version constraint, it'll pick the oldest version possible.

So if we have this constraint for instance: `symfony/symfony: ~2.3`. Normally, this will install 2.7 (or 2.8 if minimum stability is dev). With `--prefer-lowest`, it'll install 2.3.0.